content/docs/getting_started/reconciler.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
Could you please move this to `content/docs/guides/` and reference this guide in the install section on how to enable the reconciler.